deploy/runtools/firesim_topology_elements.py

```diff
@@ -546,10 +554,16 @@ def get_rootfs_name(self) -> Optional[str]:
             # cases
             return self.get_job_name() + "-" + rootfs_path.split("/")[-1]
 
-    def get_tar_name(self) -> str:
+    @classmethod
+    def get_tar_name(cls) -> str:
         """ Get the name of the tarball on the run host"""
         return "driver-bundle.tar.gz"
 
+    @classmethod
+    def get_xclbin_name(cls) -> str:
+        """ Get the name of the xclbin on the run host"""
+        return "bitstream.xclbin"
+
```

Comment:
Seems strange to me that these methods returning names relative to the `sim_slot_X` directory of a particular sim  are `@classmethod` of the `FireSimServerNode` class.

Reply:
Ok, I wasn't exactly sure where to put these